crates/biome_cli/src/execute/traverse.rs

```diff
@@ -246,10 +247,10 @@ struct DiagnosticsPrinter<'ctx> {
     ///  Execution of the traversal
     execution: &'ctx Execution,
     /// The maximum number of diagnostics the console thread is allowed to print
-    max_diagnostics: u16,
+    max_diagnostics: u64,
```

Comment:
Why did you change the type? `u32` could be enough?

Reply:
I had a casting issue among numbers